packages/test/test-end-to-end-tests/src/test/rootDatastores.spec.ts

```diff
@@ -335,7 +334,7 @@ describeFullCompat("Named root data stores", (getTestObjectProvider) => {
 			const dataObject3 = await requestFluidObject<ITestFluidObject>(container3, "/");
 
 			await provider.ensureSynchronized();
-			assert.ok(await getRootDataStore(dataObject3, alias));
+			assert.ok(await getRootDataStoreCompat(dataObject3, alias));
```

Comment:
How did you decide where use this compat one v. the new function?

Reply:
Are we missing test coverage on the new function on account of these tests not using it?